<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Editor</class>
 <widget class="QMainWindow" name="Editor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QTreeWidget" name="projectWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <attribute name="headerVisible">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Project</string>
       </property>
      </column>
      <item>
       <property name="text">
        <string>Create new or load existing</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="sceneTabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="sceneTab">
       <attribute name="title">
        <string>Scene View</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="sceneVerticalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QToolBar" name="sceneToolBar">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="windowTitle">
             <string>toolBar</string>
            </property>
            <property name="iconSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
            <addaction name="select_none"/>
            <addaction name="select_traslate"/>
            <addaction name="select_scale"/>
            <addaction name="select_rotate"/>
           </widget>
          </item>
          <item>
           <widget class="QOpenGLWidget" name="openGLWidget">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="exit"/>
   </widget>
   <widget class="QMenu" name="menuProject">
    <property name="title">
     <string>Project</string>
    </property>
    <addaction name="new_project"/>
    <addaction name="open_project"/>
    <addaction name="save_project"/>
    <addaction name="close_project"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="options"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuProject"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="select_none">
   <property name="icon">
    <iconset resource="editor.qrc">
     <normaloff>:/editor/icons/icons/mouse-pointer.svg</normaloff>:/editor/icons/icons/mouse-pointer.svg</iconset>
   </property>
   <property name="text">
    <string>None</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="select_traslate">
   <property name="icon">
    <iconset resource="editor.qrc">
     <normaloff>:/editor/icons/icons/move.svg</normaloff>:/editor/icons/icons/move.svg</iconset>
   </property>
   <property name="text">
    <string>Translate</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="select_scale">
   <property name="icon">
    <iconset resource="editor.qrc">
     <normaloff>:/editor/icons/icons/maximize-2.svg</normaloff>:/editor/icons/icons/maximize-2.svg</iconset>
   </property>
   <property name="text">
    <string>Scale</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="select_rotate">
   <property name="icon">
    <iconset resource="editor.qrc">
     <normaloff>:/editor/icons/icons/rotate-cw.svg</normaloff>:/editor/icons/icons/rotate-cw.svg</iconset>
   </property>
   <property name="text">
    <string>Rotate</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="new_project">
   <property name="text">
    <string>New Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="open_project">
   <property name="text">
    <string>Open Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="exit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="save_project">
   <property name="text">
    <string>Save Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="close_project">
   <property name="text">
    <string>Close Project</string>
   </property>
  </action>
  <action name="options">
   <property name="text">
    <string>Options</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="editor.qrc"/>
 </resources>
 <connections/>
</ui>
